eslint.config.mjs

```diff
@@ -349,6 +350,7 @@ export default tseslint.config(
     rules: {
       "@typescript-eslint/no-confusing-void-expression": "off",
       "@typescript-eslint/no-magic-numbers": "off",
+      "no-undefined": "off",
```

Comment:
why is it "off" here but "error" above?

Reply:
I've switched this rule off for the unit tests. We need to be able to use `undefined` within these to exercise all edge cases, so I deemed this was the best solution.

Unfortunately, the diff doesn't make this clear. But this set of rules has a glob to only specify test files.